packages/teleport/src/services/mfa/types.ts

```diff
@@ -0,0 +1,15 @@
+export type MfaDevice = {
+  id: string;
+  type: 'Hardware Key' | 'Authenticator App';
```

Comment:
i'm not really seeing the benefit of typing this, do we use this else where other than in `makeMfaDevice`? If not, i think it's better to just contain it inside the makeXXX?

Reply:
We do use it in a couple places in `webapps.e`

Reply:
this is a strange name for a type, please remove any spaces

Reply:
@alex-kovoy this is just for the text on the `type` column in the table, I will rename `type` to `typeLabel` to make it more clear